scripts/hostcfgd

```diff
@@ -76,6 +77,38 @@ CFG_DB = "CONFIG_DB"
 STATE_DB = "STATE_DB"
 
 
+def get_salt():
+    file_path = "/etc/shadow"
+    target_username = "admin"
```

Comment:
this variable never used, can be cleaned up

Reply:
sure